src/provider/cloud-element-templates/ElementTemplatesPropertiesProvider.js

```diff
@@ -44,8 +43,8 @@ export default class ElementTemplatesPropertiesProvider {
         label: 'Template',
         component: createElementTemplatesGroup({
           getTemplateId,
-          unlinkTemplate,
-          updateTemplate
+          unlinkTemplate: this._elementTemplates.elementTemplates,
```

Comment:
Let's make this an arrow function instead? 

```javascript
unlinkTemplate: (...args) => this._elementTemplates.unlinkTemplate(...args)
```

I feel this is easier to understand; also I don't want us to encorage "execute as functions" on instances.

Also, this is likely a typo (cf. `_elementTemplates.elementTemplates`) and not going to work. Please test :slightly_smiling_face:

Reply:
I know why it didn't. We don't actually need that anymore - will follow up

Reply:
Also, as we have the `useService` pattern, we may use it here, too.